packages/platform-sdk/test/integration-tests/test-utils.ts

```diff
@@ -57,6 +56,18 @@ export const authMiddlewareOptions = {
   fetch,
 }
 
+export const authMiddlewareOptionsV3 = {
+  host: authURL,
+  projectKey,
+  credentials: {
+    clientId: clientId,
+    clientSecret: clientSecret,
+  },
+  tokenCache,
+  scopes: [`manage_project:${projectKey}`],
+  httpClient: fetch,
```

Comment:
I guess the type is only different because of this httpClient was missing? In this case it actually should get `fetch` as the default httpClient: https://github.com/commercetools/commercetools-sdk-typescript/blob/97fac5f63e4a3b250f0818a3ee2ede95f612d9da/packages/sdk-client-v3/src/client/builder.ts#L108. Maybe this should be fixed as well?

Reply:
The one in the test just needs that `httpClient` property whilst that in the builder is fine, it doesn't need fixing. As you can see we separated it into `authMiddlewareOptions` for v2 and v3

Reply:
I think it is not fine because you wrote above that the tests were failing because of this. But in this case it should take the default fetch and not fail when you don‘t provide `httpClient`.

Reply:
No, I think you are getting it wrong. this is not specifically why the test is failing. Rather `expect(response.originalRequest.headers['User-Agent']).toContain(...)` from [this PR](https://github.com/commercetools/commercetools-sdk-typescript/pull/708), please take a look at [this](https://app.circleci.com/pipelines/github/commercetools/commercetools-sdk-typescript/4036/workflows/8197356e-e9f1-4873-8583-a92296e7fbf7/jobs/8925?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link&utm_content=summary) to see why it is failing.

Also, I know we defaulted it, tests are also meant to show how to use the SDK so removing `httpClient` property entirely or using a value (in this case `fetch`) which will result in an undefined value at `options.httpClient` makes not sense.

Reply:
Oh I see, thanks for explaining.